elastic/kibana
# [Console] Only `copy as curl` should be allowed for kbn requests

## Summary

request-converter has recently released [a patch](https://github.com/elastic/request-converter/issues/71) that allows consumers to copy kbn requests as curl. This PR adds support in kibana for that, and throws a warning message when the user tries to copy a kbn request as anything else than curl. 


#### How to test

Make sure that kbn requests can be copied as curl, but not as any other languages (also check for multiple requests).
